_posts/2021-02-28-openttd-1-11-0-beta2.md

```diff
@@ -0,0 +1,53 @@
+---
+title: OpenTTD 1.11.0-beta2
+author: TrueBrain
+---
+
+Today we release our second beta of our 1.11 release series!
+
+This second beta is a huge one: we spend the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
+With a huge thanks to michi_cc, we now even have OpenGL support!
+Don't get your hopes up: it still is (and always will be) a 2D game.
+
+To make sure these changes get their fair share of testing, we ask you to help us test them. Especially the following new features:
+* OpenGL support for all platforms.
+* Massive improvements to all video-drivers; so massive, we now have a setting to indicate an upper-limit to fast-forward speeds.
+* Display now runs on 60fps.
```

Comment:
"or higher, if you alter the refresh-rate setting in the config file" ?

Reply:
Let's for now keep this a secret :)